FAQ.md

```diff
@@ -1,20 +1,129 @@
+
+## Governance structure of the foundation
+
+<a id="q-bylaws"></a>
+**🤔 Q: How will the foundation structure its board, bylaws and operations?**
+
+💁 A: We’re still finalizing all the details around the foundation, and we plan to have the legal documents approved by the end of the year. Once that happens we’ll announce everything around how the foundation works.
+
+Still, there are a few things we can say now:
+
+* The board will have representatives from both sponsor companies and project directors.
+* The project directors will be drawn from active Rust team members.
+* In true Rust spirit, we are structuring things so as to encourage active collaboration between the sponsor representatives and the project directors, and to ensure that the project directors are on board with whatever actions the Foundation takes.
+
+Something else that’s very important to us: we want the workings of the Foundation to be accessible to everyone. To that end, we are planning to release not only the “legal bylaws” but also “human readable” versions (translated into multiple languages), as well as “guidebook”-like materials to help folks understand what it means to be a part of the foundation.
+
+<a id="q-individual-donations"></a>
+**🤔 Q: As an individual, can I donate money to the Rust Foundation?**
+
+💁 A: Not yet but perhaps in the future. To start, we explicitly prioritized getting donations from companies over individuals. We believe that individuals -- especially those currently contributing to the Rust project -- already give so much to the project. While we have a number of corporate sponsors for Rust's infrastructure already, we have not been able to sponsor the Rust project’s broader needs in a directed manner quite yet. We want companies to take responsibility for giving back to the Rust community and project for the value that volunteers bring; their sponsoring the Rust Foundation is one key way for them to do so. The question on [hiring contributors](#q-hiring) also talks about some of the other ways we’re encouraging investment in the Rust project from our corporate sponsors.
+
+<a id="q-individuals"></a>
+**🤔 Q: Will Rust maintainers have a voice in the foundation?**
+
+💁 A: Yes, absolutely Rust maintainers have a voice. The Foundation board will have a number of seats that are occupied by Rust team members and who are tasked with representing the project. Moreover, the by-laws ensure that those directors must agree with whatever the Foundation does.
```

Comment:
> Moreover, the by-laws ensure that those directors must agree with whatever the Foundation does.

I think we should change this "all decisions will require support from those directors and the directors appointed by the sponsors".